docs/03-extensions/stripe/index.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: "Stripe"
+description:
+  "This extension allow the usage of Stripe in your Front-Commerce application."
+---
+
+<SinceVersion tag="3.5" />
+
+<p>{frontMatter.description}</p>
+
+## Installation
+
+First ensure you have installed the package:
+
+```bash
+$ pnpm install @front-commerce/stripe
+```
+
+## Front-Commerce configuration
+
+[After installing the Stripe package](/docs/3.x/extensions/stripe/#installation),
+you need to enable the corresponding extension. For that, you need to tweak your
+`front-commerce.config.ts` file like this:
+
+```typescript title="front-commerce.config.ts"
+import { defineConfig } from "@front-commerce/core/config";
+import themeChocolatine from "@front-commerce/theme-chocolatine";
+import magento2 from "@front-commerce/magento2";
+// highlight-next-line
+import stripe from "@front-commerce/stripe";
+import storesConfig from "./app/config/stores";
+import cacheConfig from "./app/config/caching";
+
+export default defineConfig({
+  extensions: [
+    magento2({ storesConfig }),
+    themeChocolatine(),
```

Comment:
// ⚠️ needs to be before magento2() 😇 see https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/b9840a77f327f56d562fea807357d62f6243939c/skeleton/front-commerce.config.ts#L16-21

Reply:
fixed